.travis.yml

```diff
@@ -3,11 +3,11 @@ dist: focal
 language: node_js
 
 node_js:
-  - 14.21.2
+  - 16.20.1
 
 env:
   global:
-  - MONGODB=5.3.2
+  - MONGODB=6.0.10
```

Comment:
Does this match our current production version? If so, I'll update the repos I updated to this version, too.

Reply:
I updated this based on from your question  in the backed channel. Jimmy said the current Mongo version was `6.0.10`